region/norfair/east/Upper Norfair Farming Room.json

```diff
@@ -250,28 +267,24 @@
       "name": "Base",
       "notable": false,
       "requires": [
-        "h_canNavigateHeatRooms",
-        {"heatFrames": 120},
-        {"or": [
-          {"and": [
-            {"heatFrames": 30},
-            "Wave"
-          ]},
-          {"obstaclesCleared": ["A"]}
-        ]}
-      ],
-      "clearsObstacles": ["A"]
+        {"heatFrames": 90},
+        {"obstaclesCleared": ["A"]}
+      ]
     },
     {
       "link": [5, 2],
-      "name": "Gate Glitch",
+      "name": "Leave with Runway",
       "notable": false,
       "requires": [
-        "h_canNavigateHeatRooms",
-        {"heatFrames": 120},
-        "h_canHeatedBlueGateGlitch"
+        {"heatFrames": 90},
```

Comment:
Is this right? 5->2 takes 90 frames, and 5-> the beginning of the runway takes the same